test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -3014,13 +3014,15 @@ TEST_F(AtenXlaTensorTest, TestBitwiseNotInPlace) {
 
 TEST_F(AtenXlaTensorTest, TestSign) {
   torch::Tensor a =
-      torch::randn({2, 2}, torch::TensorOptions(torch::kFloat)) * 100.0;
+      torch::randn({200, 200}, torch::TensorOptions(torch::kFloat).requires_grad(true));
```

Comment:
one annoying thing - the test breaks with an autograd error if we do `a = torch.tensor(..., requires_grad=true) * 100`. I'm not totally sure what the autograd interaction is, but it looks like the reference cpu tensor needs to be created in one shot. Maybe we could avoid that by doing `.detach().set_requires_grad(true)`, but that felt clunkier.

Reply:
I think the purpose of *100 is to get the random number within (-100, 100) instead of (-1, 1). Would do a inplace mul fix the problem here?

Reply:
I just read the test way too quickly 😛 yeah, that should probably work. I'll fix it

Reply:
fixed, thanks. I had to set `requires_grad=true` after the inplace update, since autograd doesn't handle leaf mutations.